### ISSUE TITLE
Missing Galician translation

### ISSUE THREAD

**Issue by jbrazio** [MarlinFirmware/Marlin#3357]
We need your contribution to translate the following missing messages:

``` cpp
#define MSG_LEVEL_BED_HOMING
#define MSG_SET_HOME_OFFSETS
#define MSG_BED_Z
```

Have a look on the file [language_en.h](https://github.com/MarlinFirmware/Marlin/blob/RC/Marlin/language_en.h) for the original English translation.

**Comment by jbrazio** [comment]
Closed by #3772.

**Comment by jbrazio** [comment]
I've submitted a PR to prune unmaintained language files, @rafacouto as you're one of the contributors to this file would you care to submit an update to it ? Thanks.
